cmd/user-agent/src/pages/webWalletSamples.js

```diff
@@ -99,6 +105,41 @@ const udc = {
     }
 }
 
+const govnVC = {
+    "@context": ["https://www.w3.org/2018/credentials/v1", "https://trustbloc.github.io/context/governance/context.jsonld", "https://trustbloc.github.io/context/vc/examples-v1.jsonld"],
+    "credentialStatus": {
+        "id": "http://governance.vcs.example.com:8066/governance/status/1",
+        "type": "CredentialStatusList2017"
+    },
+    "credentialSubject": {
+        "data_uri": "https://example.com/data.json",
+        "define": [{
+            "id": "did:trustbloc:testnet.trustbloc.local:EiDniKF0RDQVRuCSwi7N87O-x7axF7bUZ9tA12uq4qiWLQ",
+            "name": "DID"
+        }],
+        "description": "Governs accredited financial institutions, colleges and universities.",
+        "docs_uri": "https://example.com/docs",
+        "duties": [{"name": "safe-accredit", "uri": "https://example.com/responsible-accredit"}],
+        "geos": ["Canadian"],
+        "jurisdictions": ["ca"],
+        "logo": "https://example.com/logo",
+        "name": "Trustbloc Govn",
+        "privileges": [{"name": "accredit", "uri": "https://example.com/accredit"}],
+        "roles": ["accreditor"],
+        "topics": ["banking"],
+        "version": "1.0"
+    },
+    "issuer": "did:trustbloc:testnet.trustbloc.local:EiDdRGN4x2S4D0xYTPaJEHcD50Sq5fgv0sUfbdgY7x6lkQ",
```

Comment:
In a followup, we will need to be more sophisticated about which issuers and credentials are acceptable for this purpose.

Reply:
Will add it in follow up issue.